src/endpoints/load_balancing/pool_details.rs

```diff
@@ -0,0 +1,23 @@
+use crate::endpoints::load_balancing::Pool;
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// Pool Details
+/// https://api.cloudflare.com/#account-load-balancer-pools-pool-details
+pub struct PoolDetails<'a> {
+    /// The Cloudflare account to create this Pool under.
```

Comment:
I don't think this comment is accurate here –– maybe "The Cloudflare account of this pool"?

Reply:
Good points, fixed!